acceptance/bundle/deploy/lakebase/database-instance/single-instance/script

```diff
@@ -0,0 +1,16 @@
+envsubst < databricks.yml.tmpl > databricks.yml
+
+cleanup() {
+   trace $CLI bundle destroy --auto-approve
+}
+trap cleanup EXIT
+
+trace $CLI bundle validate
+
+# URL is excluded because some cloud envs add an ?0= query param and some don't
+trace $CLI bundle summary | grep -v "URL"
```

Comment:
I think before deploying it would `(not deployed)` anyway, right? Shall we leave it in this case?
Also we can add REPL to remove `?o=`

Reply:
1. I see that summary returns an URL even if the resource is not deployed yet, is this correct?
2. The problem with `?o=` is that it is present in some clouds and is absent in others. Is there a way to solve this with a replacement?

Reply:
1. No, not really, it should return `(not deployed)` status.
2. I think it should be possible to add `?o=(...)` regexp to match it and replace with empty string?